### PR TITLE
Correct overrided qa message

### PR DESCRIPTION
**Pull request by gaborod16**
### Bug Fixes

* QA Response. The original QA response is no longer overridden by the QA response to track in the CAI.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
+* QA Response. The original QA response is no longer overridden by the QA response to track in the CAI.
+
+*****************
+
 ## Release ONDEWO BPI v4.1.0
 
 ### Improvements
 
-* Extend the Q&A BPI version to support filtering functionality through context injection
+* Extend the Q&A BPI version to support filte<ring functionality through context injection
```
Comment (kNalj) [inline RELEASE.md]: You fat fingered a less then in the middle of the word.

File: ondewo_bpi_qa/bpi_qa_server.py [hunk 2]
```diff
@@ -340,13 +353,13 @@ def _fill_in_qa_response_with_cai_response(self, qa_response, cai_response) -> G
                                     postback=f'{_fm_button.open_uri_action.uri}'
                                 )
                             ],
-                            image_uri='https://vera.wkooe.at/ki/embed/img/anpassungen/chat/chatleiste_vera_neu.svg',
+                            image_uri=QA_PUBLIC_IMAGE_URI_LINK,
```
Comment (kNalj) [inline ondewo_bpi_qa/bpi_qa_server.py:356]: :art:

File: ondewo_bpi_qa/bpi_qa_server.py [hunk 3]
```diff
@@ -295,32 +299,41 @@ def create_session_if_not_exists(self, request: DetectIntentRequest, ) -> None:
...
+            self,
+            qa_response: GetAnswerResponse,
+            cai_response: DetectIntentResponse
+    ) -> GetAnswerResponse:
+        """
+        This function updates the QA response with information from the CAI response.
+        The objective is to create a QA response that can be added to the Session Review History.
 
-            Note: this function modifies the qa_response directly.
+            -> Returns a copy of the original qa_response with the modified fields.
```
Comment (kNalj) [inline ondewo_bpi_qa/bpi_qa_server.py:311]: :art:

**Review by kNalj — APPROVED** [review]
Just fix the typo.

File: RELEASE.md (hunk 1 above)
Comment (gaborod16) [reply RELEASE.md]: I fixed it, you needed to refresh the page XD

**Pull request merged**